### PR TITLE
#420 Adding canonical refs to ./getting-started part 2/10

### PR DESCRIPTION
**Pull request by martyav**
See https://github.com/rancher/rancher-docs/pull/619 for more info

### CODE REVIEW

File: docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 title: Feature Flags
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades"/>
```
Comment (btat) [inline docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/feature-flags"/>
```

File: versioned_docs/version-2.0-2.4/reference-guides/installation-references/feature-flags.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 title: Feature Flags
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades"/>
```
Comment (btat) [inline versioned_docs/version-2.0-2.4/reference-guides/installation-references/feature-flags.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/feature-flags"/>
```

File: versioned_docs/version-2.5/reference-guides/installation-references/feature-flags.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 title: Feature Flags
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades"/>
```
Comment (btat) [inline versioned_docs/version-2.5/reference-guides/installation-references/feature-flags.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/feature-flags"/>
```

File: versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-references/feature-flags.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 title: Feature Flags
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades"/>
```
Comment (btat) [inline versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-references/feature-flags.md:6]:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/feature-flags"/>
```

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-references/feature-flags.md (hunk 4 above)
Comment (martyav) [reply versioned_docs/version-2.6/getting-started/installation-and-upgrade/installation-references/feature-flags.md]: Just pushed [b1d6243](https://github.com/rancher/rancher-docs/pull/621/commits/b1d62436c5df2ebb861fbded6d760f2822213aca) to correct the copy/paste error

**Review by btat — APPROVED** [review]

**Pull request merged**